walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -20,25 +20,27 @@
 import java.util.Arrays;
 
 public final class FindMeetingQuery {
-  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    
-    //initialize availibility collection
     Collection<TimeRange> availibility;
-    
-    
+  public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
+
     //no attendees -- first test
-    if (events == null){
+    if (request.getAttendees() == null){
```

Comment:
not sure why first test fails ... I think I'm checking/returning right thing

Reply:
After you do this check and you find out that the whole day is available, you shouldn't have to check for the rest of the code below (specifically the part that iterates the events).

The test is failing because after you set your return set to WHOLE_DAY, something else is modifying it again.